.gitignore

```diff
@@ -43,6 +43,7 @@ dist/*
 _notes
 _notes/*
 Thumbs.db
+composer.lock
```

Comment:
The lock file is important to ensure the built copy of Gravity PDF exactly matches what we run in development. My previous comment about not including it in your PR was because it was unrelated to the work you are doing. We try keep our PRs to one specific focus. In this case, refactoring the unit tests.

Reply:
Noted Jake.